homeassistant/components/yolink/manifest.json

```
@@ -0,0 +1,10 @@
+{
+  "domain": "yolink",
+  "name": "YoLink",
+  "config_flow": true,
+  "documentation": "https://www.home-assistant.io/integrations/yolink",
+  "requirements": ["paho-mqtt==1.6.1", "yolink-client==0.0.8"],
+  "dependencies": ["http"],
```

This changed recently.
```suggestion
  "dependencies": ["auth"],
```